web-client/src/components/StyledComponents/FlexBox.tsx

```diff
@@ -0,0 +1,41 @@
+import styled  from 'styled-components';
+interface IFlexBoxProps {
+  justify?: string;
+  direction?: string;
+  align?: string;
+  width?: string;
+  maxWidth?: string;
+  height?: string;
+  margin?: string;
+  padding?: string;
+  backgroundColor?: string;
+  borderRadius?: string;
+  textAlign?: string;
+  scrollY?: string;
+  flex?: string;
+  deskWidth?: string;
+}
+  // flex-shrink: 0;
+
+const FlexBox = styled.div<IFlexBoxProps>`
```

Comment:
there is the row-gap and column-gap property. I think it would be a great addition to this component: https://css-tricks.com/almanac/properties/g/gap/

Reply:
If I'm not mistaken, these are properties use with display:grid?  but this component has display:flex

Reply:
No no row-gap and column-gap are for display:flex it is super helpful. Since I know it exists I use it all the time